src/test/java/com/company/tree/Helper.java

```diff
@@ -0,0 +1,14 @@
+package com.company.tree;
+
+public class Helper {
```

Comment:
Rename the class name with appropriate terms.

Reply:
this method is written to assert in test